Jenkinsfile

```diff
@@ -1,5 +1,4 @@
 buildPlugin(configurations: [
-  [ platform: "linux", jdk: "8", jenkins: null ],
-  [ platform: "windows", jdk: "8", jenkins: null ],
-  [ platform: "linux", jdk: "11", jenkins: null ]
+  [platform: "linux", jdk: 11],
+  [platform: "windows", jdk: 11]
```

Comment:
Optional feedback for your consideration: Adding a Java 17 build here would be forward-thinking. Although this plugin is tested with Java 17 in BOM/PCT tests, testing it with Java 17 in the plugin's `Jenkinsfile` would enable compatibility problems to be detected earlier, before a plugin PR is integrated.

Reply:
Good point, I can set that up.